.buildkite/bk.integration.pipeline.yml

```diff
@@ -40,8 +40,11 @@ steps:
         matrix:
           - default
           - fleet
+          - fleet-endpoint-security
```

Comment:
The ordering in these matrix lists looks arbitrary. Is there a reason for the ordering, or should we have them sorted alphabetically?

Reply:
Honestly, there are no ordering rules. Just placed fleet groups together

Reply:
Are you going to sort them? If the order doesn't matter, I think it'd be a better user experience when engineers are scrolling through the results of a BK job to have them alphabetical. Plus it would make these lists more easily maintainable (until we figure out making the groups/matrix dynamic).